CodexNetDeployer/Deployer.cs

```diff
@@ -45,19 +45,16 @@ public CodexDeployment Deploy()
 
             // Each node must have its own IP, so it needs it own pod. Start them 1 at a time.
             var codexStarter = new CodexNodeStarter(config, lifecycle, gethResults, config.NumberOfValidators!.Value);
-            var startResults = new List<CodexNodeStartResult>();
+            var codexContainers = new List<RunningContainer>();
             for (var i = 0; i < config.NumberOfCodexNodes; i++)
             {
-                var result = codexStarter.Start(i);
-                if (result != null) startResults.Add(result);
+                var container = codexStarter.Start(i);
+                if (container != null) codexContainers.Add(container);
             }
 
-            var (prometheusContainer, grafanaStartInfo) = StartMetricsService(lifecycle, setup, startResults.Select(r => r.Container));
+            var (prometheusContainer, grafanaStartInfo) = StartMetricsService(lifecycle, setup, codexContainers);
 
-            CheckPeerConnectivity(startResults);
-            CheckContainerRestarts(startResults);
```

Comment:
There seem to be quite a few changes here that are not related to the annotations or pod labels. It looks like a merge/conflict resolution didn't go so well. Please check.